executor.go

```diff
@@ -33,7 +33,7 @@ func Execute(p ExecuteParams) (result *Result) {
 		ctx = context.Background()
 	}
 	if p.manager == nil {
-		p.manager = manager
```

Comment:
This is messing with my mind -- where did this old value even come from that it worked?

Reply:
Whoops I thought I was fixing this for a test, I shouldn't have made that change in the main executor. I'm glad you pointed it out. To answer your question though it comes from https://github.com/GannettDigital/graphql/blob/master/graphql.go

Reply:
Ahh, the `var (` being on a different line defeated my `git grep`-Fu by looking like a function use.